### DISCUSSION TITLE
PHPMailer are sending a lot of emails (thousands per day)

### DISCUSSION THREAD

**Discussion by guilhermeal** [PHPMailer/PHPMailer#2542] [General]
I don't know, if here is the best place to put my question/problem. If not, please, sorry; Can you show me a way?

I was using Phpmailer 6.2.0 when I detect this problem. 
My SMTP are AWS SES and this, sendend thousands mails generating a high price on my invoice. I think that are SPAMs.. But. I don't know really.

So, 

I seen that release 6.5.0 solve some problems about SECURITY that may permit untrusted code to be run from an address validator... And, I updated my enviroment to 6.5.1... 

But that don't solve my problem.. My server still sending a lot of mails, around 20k per day.

I had have inactive my IAM Credentials to stop this sends..

I don't know whats happens..
Who can help me?

Thank you..

**Comment by Synchro** [comment]
It's most likely that how you are *using* PHPMailer is the problem. If for example you have a contact form that allows a user to send an arbitrary message to a user-supplied address, it can be used as a spam gateway. Check how you're using PHPMailer.

**Reply by guilhermeal** [reply]
I don't use directly... I use PHPMailer, after a signup, for example... After submit, the script php check data, checking for entry if they have sql injections, save in DB and, if all are right, will send the mail to new user.

**Reply by Synchro** [reply]
So add logging to all your mail functions to check what is being run. It's likely that every outbound message will be triggered by an earlier HTTP request - track down where they connect up.
